### ISSUE TITLE
G'MIC install for GIMP 2.10 on Mac OS High Sierra

### ISSUE THREAD

**Issue by ElToune** [dtschump/gmic#142]
Hi guys,

Installed GIMP 2.10 and downloaded G'MIC but can't install it correctly, there is no readme.txt and I looked all over the place but can't find any explanation of the install process (the old process described here: https://itstillworks.com/edit-dds-files-gimp-7343632.html doesn't seem valid anymore, as the mentioned plug-in folder doesn't exist on my computer (cf. attached screenprint).

Did anyone successfully install G'MIC for GIMP 2.10 on Mac OS and could help me with that?

Thank you!!
<img width="1187" alt="Library - App Support - GIMP" src="https://user-images.githubusercontent.com/52702326/60883952-e59c8d80-a24b-11e9-97ca-692740898042.png">

**Comment by DougStoker** [comment]
make a folder named plug-ins in the 2.10 folder and then move the stuff in the gmic folder to it

**Comment by dtschump** [comment]
@karo11, any idea?

**Comment by karo11** [comment]
Hi David,
I gave a slightly, hopefully, better comprehensible, receipt, at least for
the cli. Still I did not see any feedback!

I think the Andrea build for gmic as well as gmic-qt is functional for High
Sierra!

The user tried to get info on different channels, maybe he/she did not
look?

Possibly the user tried the Partha build. Still Partha seems to have
difficulties with Gimp 2.10.12 AND with g'mic.

David Tschumperlé <notifications@github.com> schrieb am Di., 6. Aug. 2019,
12:10:
